modules/products/bitbucket/helm.tf

```diff
@@ -43,6 +43,11 @@ resource "helm_release" "bitbucket" {
           persistentVolumeClaim = {
             create = true
           }
+          resources = {
```

Comment:
`resources` should be within `persistentVolumeClaim`, not same level. 
Same for Confluence.

Reply:
Jira as well

Reply:
and other products